src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -46,16 +42,17 @@ const BasicCard = (props: BasicCardProps) => {
                 >
                     <Content
                         title={title}
-                        titleId={titleId}
                         text={text}
-                        textId={descriptionId}
                         additionalInfo={additionalInfo}
                         links={links}
                         list={list}
                         buttons={buttons}
                         size="s"
                         colSizes={{all: 12, md: 12}}
                         controlPosition={areControlsInFooter ? 'bottom' : 'default'}
+                        a11yProps={{
+                            title: {role: 'link', tabIndex: 0},
```

Comment:
If you want `link` role, it is better to use `<a>` tag: no need to specify the `role` and `tabIndex`. But be aware of layout change due to browser default styles